### PR TITLE
Small tweaks to k8s container registry documentation and test implementation

### PR DESCRIPTION
**Pull request by supertetelman**
* move the deployment from k8s-cluster.yml and into the test script
* Fixe broken docs/links.

### CODE REVIEW

File: docs/k8s-cluster/README.md
Comment (ajdecon) [inline docs/k8s-cluster/README.md:198]: I think this should still be yml? :)

**Comment by supertetelman** [comment]
Do not merge this until the helm upgrade merge goes through. It is dependant on the functioning of "--create-namespace".

**Review by michael-balint — APPROVED** [review]

**Pull request merged**
